src/main/java/webserver/RequestHandler.java

```diff
@@ -42,13 +32,13 @@ public void run() {
             HttpRequest httpRequest = HttpRequest.of(br);
             HttpResponse httpResponse = new HttpResponse();
             if (httpRequest.isStaticFileRequest()) {
-                httpResponse.response(httpRequest);
+                HttpResponseProcessor.init(httpRequest, httpResponse);
             } else {
-                Controller controller = controllers.get(httpRequest.getHost());
+                Controller controller = ControllerInitializer.get(httpRequest.getHost());
```

Comment:
host 보다는 path 라는 네이밍이 더 적절해 보이네요 :)
[URL과 리소스 [HTTP 완벽가이드]](https://futurists.tistory.com/95)
```suggestion
                Controller controller = ControllerInitializer.get(httpRequest.getPath());
```